app/controllers/gobierto_indicators/indicators_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module GobiertoIndicators
+  class IndicatorsController < GobiertoIndicators::ApplicationController
+    include User::SessionHelper
+
+    before_action :load_year, only: [:ita, :ip]
+
+    def index
+      redirect_to gobierto_indicators_ita_path
+    end
+
+    def ip
+    end
+
+    def ita
+      @ita_json = "/generated.json"
+    end
+
+    def gci
+    end
+
+    private
+
+    def load_year
```

Comment:
Too many things for a method called `load_year`, because it's also doing the redirection to the indicator.